tests/unit/ListPeopleAndRoles.spec.ts

```diff
@@ -0,0 +1,244 @@
+// Libraries
+import Vue from 'vue'
+import Vuelidate from 'vuelidate'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import mockRouter from './MockRouter'
+
+// Store
+import { store } from '@/store'
+
+// Utils
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
+
+// Components
+import { ListPeopleAndRoles } from '@/components/AddPeopleAndRoles'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+
+let vuetify = new Vuetify({})
+
+describe('List People And Roles', () => {
+  let wrapperFactory: any
+
+  const mockPersonList = [
+    {
+      'id': 0,
+      'firstName': 'Cameron',
+      'lastName': 'Bowler',
```

Comment:
Forever immortalized, haha!

Reply:
hahah... put down the first name that came into mind!